scripts/exgfs_wave_prdgen_bulls.sh

```diff
@@ -154,12 +154,12 @@
      echo "${msg}"
      set_trace
      echo "${RUN} wave prdgen ${date} ${cycle} : Missing bulletin data."
-     err=4;export err;err_chk
-     exit ${err}
+     export err=4
+     err_chk
```

Comment:
Similar question, should the `msg` on line 147 and/or `echo` on line 156 be added to `err_chk`? Also, should lines 146 to 156 be compressed to just issue the fatal error to `err_chk`?